tests/agent_features/test_configuration.py

```diff
@@ -380,108 +354,99 @@ def test_delete_setting_parent():
 #
 #       'value' == 'default'
 #       'value' != 'default'
-TSetting = collections.namedtuple('TSetting', ['name', 'value', 'default'])
+TSetting = collections.namedtuple("TSetting", ["name", "value", "default"])
 
 translate_settings_tests = [
     (
-        TSetting('transaction_tracer.capture_attributes', True, True),
-        TSetting('transaction_tracer.attributes.enabled', False, True)
+        TSetting("strip_exception_messages.whitelist", [], []),
+        TSetting("strip_exception_messages.allowlist", "non-default-value", []),
     ),
     (
-        TSetting('transaction_tracer.capture_attributes', False, True),
-        TSetting('transaction_tracer.attributes.enabled', True, True)
+        TSetting("strip_exception_messages.whitelist", "non-default-value", []),
```

Comment:
```suggestion
        TSetting("strip_exception_messages.whitelist", ["non-default-value"], []),
```

Let's make these 2 into lists to make sure the data types work